CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ This file follows the best practices from [keepachangelog.com](http://keepachang
 
 ### Fixed
 
+- Fix optional dependency issue on Linux with fsevents [#2161](https://github.com/sharetribe/sharetribe/issues/2161)
```

Comment:
6.3.0 is already released so this line should go under the next "unreleased" version. Line 19 in this file.